WS-556: Add Viewability tracking to PV modal (#13064)
bbc/simorgh: src/app/hooks/useViewTracker/README.mdx

| preventNavigation | boolean | no       | Use this if you need to perform any additional tasks after sending the click event by setting to \`true\` and awaiting the event handler callback. Ensure you redirect the user to their destination when you are done                                                                                                                                                          |
| optimizely        | object  | no       | Need to provide this prop if event tracking is needed for Optimizely, the standard method is to consume the Optimizely context via React's \`useContext\` and imported from the Optimizely SDK \`import { OptimizelyContext } from '@optimizely/react-sdk';\`, the HOC \`withOptimizelyProvider\` needs to be located higher up in the component tree as well for this to work. |

> | alwaysInView      | boolean | no       | Use this to track components that are always in view and such do not be observed by the IntersectionObserver. This flag is use as an alternative to `eventSent` state variable which is used to track components that require the use to scroll into view.                                                                                                                      |

Reviewer: ```suggestion
| alwaysInView      | boolean | no       | Use this to track components that are always in view and should not be observed by the IntersectionObserver. This flag is used as an alternative to `eventSent` state variable which is used to track components that require the use of scroll into view.                                                                                                                      |
```
